### ISSUE TITLE
Eye of Killrog

### ISSUE THREAD

**Issue by n-ghost** [dalaranwow/dalaran-wow#2159]
http://www.wowhead.com/spell=126#comments

There is a couple of issues with this warlock spell, some of them are high to critical.

1) Eye has it's own action bar that replaces your regular pet's one. When the Eye despawns, it's action bar is gone, but your pet demon action bar won't come back either (at least on Bartender4), so you're stuck without it until you summon your next demon.
2) You learn this ability at level 22. They Eye you summon somehow has 4k (4000!!!) hp, and because it can attack stuff for ~130ish damage, making crits for up to 250 (it works as intended), it gives any warlock an unspeakable advantage at lower levels, making them able to breeze through just about anything anywhere. Should be hotfixed asap, because it can turn any warlock leveling experience into joke.
3) From wowhead: 

- Casing Eye of Kilrogg will replace your current pet, causing it to temporarily despawn. How is this useful? When you cancel the effect your pet appears on top of you. This is very helpful in certain instances where you have to jump off of something and don't want your pet to path around and aggro everything nearby - set it on stay, go where you need to be, summon an Eye, and then cancel it. Your pet will teleport to you.

Currently the demon stays with you even when you channel the Eye, and can be a target for mob abilities. It's behaviour is disabled while you channel, however.

4) Any damage done to the eye should break the effect. Currently it doesn't. Should be hotfixed asap as well.

**Comment by M0dinstaller** [comment]
Is the eye supposed to be able to attack for your 5th point ?

I'll add a
6) Glyph of Kilrogg (minor) doesn't work. I tried it in shattrath and the eye couldn't fly and didn't have the 50% speed increase.

**Comment by n-ghost** [comment]
Yes, it is supposed to have an auto-attack (as per wowhead wisdom, anyway), but it should be destroyed by any attack on it as well so it wasn't that much of an issue on retail.
